website/static/css/custom.css

```diff
@@ -670,6 +674,10 @@ ul.toc-headings li a:focus {
     background-color: #313131;
 }
 
+.homeContainer .buttonWrapper a:hover {
+    border-bottom: 1px solid rgba(130,130,130,1);
```

Comment:
If we are doing 100% alpha a HEX is probably preferable.

Reply:
Done